scripts/set_deployment_project_dir.py

```
@@ -0,0 +1,103 @@
+                                task.get("profiles_dir")
+                                and "/home/ddp/clientdbts" in task.get("profiles_dir")
+                            )
+                        ):
+                            print(deployment["id"])
+else:
+    for deployment in deployments:
+        deployment_id = deployment["id"]
+        changed = False
+        if deployment.get("parameters"):
+            if deployment["parameters"].get("config"):
+                if deployment["parameters"]["config"].get("tasks"):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)